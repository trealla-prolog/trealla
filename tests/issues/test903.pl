:- initialization(main).

main :-
	catch((number_chars(N,"0_%0"),fail), _, write(ok1)), nl,
	catch((number_chars(N,"0x0_/*b"),fail), _, write(ok2)), nl.
