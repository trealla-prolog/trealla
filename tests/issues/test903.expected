ok1
ok2
